homeassistant/components/lektrico/coordinator.py

```diff
@@ -0,0 +1,51 @@
+"""Coordinator for the Lektrico Charging Station integration."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+
+from aiohttp import ClientSession
+from lektricowifi import DeviceConnectionError, lektricowifi
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import LOGGER
+
+SCAN_INTERVAL = timedelta(seconds=10)
+
+
+class LektricoDeviceDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
Can we type the DataUpdateCoordinator with the return type of the async_update_data? 


```suggestion
class LektricoDeviceDataUpdateCoordinator(DataUpdateCoordinator[...]):
```

Reply:
I changed it, thanks.